tests/execution/grpc-oneof.md

```diff
@@ -1,5 +1,11 @@
+---
+skip: true
```

Comment:
Why should this fail?

Reply:
because we need to change the returned data from grpc to either have a "type" field or `{"Type": {...}}` structure